CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 # All docs must be reviewed by the docs team.
 /docs/                                          @gatsbyjs/docs
+/contributing/                                  @gatsbyjs/docs
```

Comment:
These are actually still under docs/ right? e.g. docs/contributing is the path to the new/updated content?

Reply:
oh yes! The nested `docs` got me. Will revert!